### ISSUE TITLE
Easy question.....

### ISSUE THREAD

**Issue by Angusbrow** [stcr/subscribe-to-comments-reloaded#235]
Can this plugin also be modified to allow a notification on new posts, or is it already supposed to do this? My install does not seem to send notifications on new posts but works fine for comments . Thanks!! Angus

**Comment by Reedyseth** [comment]
Hello @Angusbrow, as for now the plugin only work with comment notification, this feature has been requested and will be add it in future releases.

Regards..!!

**Comment by Angusbrow** [comment]
Yes please, would happily donate to this :) Any suggestions as to a complimentary plugin that would do the post email? The ones Ive tried don't send a clickable link in the email, only a cold one, frustrating. Yours are always clickable, must be well written html :) Thanks again.

**Comment by Reedyseth** [comment]
> Any suggestions as to a complimentary plugin that would do the post email?

Not for now, I know there might be some out there. 

I am working in lots of patches and fixes for the new release as soon as the new version is stable I can take a look at this requirement.

**Comment by Angusbrow** [comment]
Any idea why your permalink comes to web email client hot clickable and other plug in emails show as cold, even though both in html format...?

**Comment by Reedyseth** [comment]
> Any idea why your permalink comes to web email client hot clickable and other plug in emails show as cold, even though both in html format...?

Yes, they are sending the raw email without encoding it in HTML.
